packages/linux/package.mk

```diff
@@ -103,6 +103,7 @@ PKG_AUTORECONF="no"
 
 if [ "$TARGET_KERNEL_ARCH" = "arm64" -a "$TARGET_ARCH" = "arm" ]; then
   if [ "$PROJECT" = "Switch" ]; then
+	  PKG_DEPENDS_HOST="$PKG_DEPENDS_HOST gcc-linaro-aarch64-linux-gnu:host"
```

Comment:
Could you fix the indent here ? We use two spaces

Reply:
Done 👍